contract/test-utils.ts

```
@@ -0,0 +1,413 @@
+        OPERATOR_KEY = env.PRIVATE_KEY!;
+        OPERATOR_ID = env.ACCOUNT_ID!;
+        MIRROR_NODE_URL = process.env.MIRROR_NODE_URL!
+    }
+    let tokens: { tokenId: string, balance: number }[] = [];
+    let nextLink: string | null = `/api/v1/accounts/${accountId}/balances?limit=100`;
+
+    while (nextLink) {
+        const response = await fetch(`${MIRROR_NODE_URL}${nextLink}`);
+        const data = await response.json()
+        return tokens
+        nextLink = null
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)